www/nginx/src/opnsense/mvc/app/controllers/OPNsense/Nginx/forms/upstream.xml

```diff
@@ -23,6 +23,20 @@
     <advanced>true</advanced>
     <help>If you enable the proxy protocol, an upstream proxy or server will get the client IP and the server port before the real traffic is sent.</help>
   </field>
+  <field>
+    <id>upstream.host_port</id>
+    <label>Host header port</label>
+    <type>text</type>
+    <advanced>true</advanced>
+    <help>Add this port value to the Host header. Not used by default.</help>
+  </field>
+  <field>
+    <id>upstream.xfh_unchanged</id>
+    <label>XFH unchanged</label>
```

Comment:
I don't like the name as it may be confusing.

How About "Forward original Host header"